lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -450,12 +450,9 @@ def compute_group_resource_and_labels(tag)
         # Do this here to avoid having to repeat it for each record.
         match_data = @compiled_kubernetes_tag_regexp.match(tag)
         if match_data
-          group_resource.labels['container_name'] =
-            match_data['container_name']
-          %w(namespace_name pod_name).each do |field|
-            group_common_labels["#{CONTAINER_CONSTANTS[:service]}/#{field}"] =
-              match_data[field]
-          end
+          group_resource.labels['container_name'] = match_data['container_name']
+          group_resource.labels['namespace_id'] = match_data['namespace_name']
+          group_resource.labels['pod_id'] = match_data['pod_name']
```

Comment:
@qingling128: That would involve deprecating (support for) the [kubernetes_metadata_filter](https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter) plugin. I would rather tackle that separately. As things stand, the `container_name` field from the `kubernetes` record will have been overwritten by this code already -- we're just adding a couple more.